ci/open-pullrequest.ps1

```diff
@@ -5,13 +5,12 @@ param(
 )
 
 function Get-PullRequest() {
```

Comment:
Could you please add some comments for this function? Probably it would make sense to describe interface of the result object here (like 'html_url')- so it would be easier to use further

Reply:
Done. Comment about result object is added.